docs/src/contributing.md

```diff
@@ -0,0 +1,67 @@
+# Contributing to Javis
+
+Javis is currently under heavy development as we push to a primary release version. As such, some parts of these instructions may become outdated faster than we can update them. If you encounter an error in these instructions, please open an issue and let us know. 
+
+We follow a workflow pattern that is directly inspired by the [development workflow guide](http://docs.juliaplots.org/latest/contributing/#Development-Workflow-1) found in [`Plots.jl`](https://github.com/JuliaPlots/Plots.jl). The general workflow we expect contributors to follow is as follows:
+
+## 1. Fork the repo to your account
+
+## 2. Create a branch based on what you are developing
```

Comment:
There should probably be a step:
Open an issue for the PR you want to create such that we know that someone works on it or comment on an issue you're working on. Hope you know what I mean :wink: